feeadjuster/feeadjuster.py

```diff
@@ -204,10 +237,23 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
 plugin.add_option(
     "feeadjuster-threshold",
     "0.05",
-    "Channel balance update threshold at which to trigger an update. "
+    "Relative channel balance delta at which to trigger an update. Default 0.05 means 5%. "
+    "Note: it's also fuzzed by 1.5%",
+    "string"
+)
+plugin.add_option(
+    "feeadjuster-threshold-abs",
+    "0.001btc",
+    "Absolute channel balance delta at which to always trigger an update. "
     "Note: it's also fuzzed by 1.5%",
     "string"
 )
+plugin.add_option(
+    "feeadjuster-enough-liquidity",
+    "0msat",
+    "Beyond this liquidity threshold there is no need to adjusting fees, 0 means the feature is turned off",
```

Comment:
You could point out that this also that this changes the adjustment curve to achieve this.
```suggestion
    "Beyond this liquidity do not adjust fees. "
    "This also modifies the fee curve to achieve having this amount of liquidity. "
    "Default: '0msat' (turned off).",
```